textproc/extractpdfmark/Portfile

```diff
@@ -27,23 +27,21 @@ checksums           rmd160  1f1bac62afac8b52e54a796084d5b6c87453122e \
                     sha256  63f3ababd5b1081ef92ff7a417597302327c1db3902cdb9827fade147558e6db \
                     size    295003
 
+depends_build       port:pkgconfig
+
 depends_lib         port:libiconv \
-                    port:pkgconfig \
                     port:poppler
 
 post-destroot {
-    set docdir ${prefix}/share/doc/${name}-${version}
-    set mandir ${prefix}/share/man/man1
+    # The man page gets automatically installed.
+
+    set docdir ${prefix}/share/doc/${name}
 
     xinstall -d ${destroot}${docdir}
-    xinstall -d ${destroot}${mandir}
     xinstall -m 0644 -W ${worksrcpath} \
         README.md \
         README.ja.md \
         ${destroot}${docdir}
```

Comment:
Maybe I need to check again but these two doc files were installed in my case as well.

Reply:
Mhmm, I would prefer to leave that in – a `README.ja.md` file is somewhat exotic, so explicit handling is not a bad thing IMHO.

Reply:
Except that I don't quite understand in what way is installing `README.ja.md` automatically any different from explicitly installing it again in a separate step? Or what am I missing?

From the given code my impression would be that deleting `README.ja.md` from the `Portfile` would also remove the file from installed files, but this is not the case.

Reply:
OK, forced push to follow your advice.